### PR TITLE
Created GitHub Environment for Final API Project

### PR DESCRIPTION
**Pull request by NatanyaDC**
Created API Final Project, Get file, and Post file.

### CODE REVIEW

**Comment by NatanyaDC** [comment]
Changes made!

File: student-work/2023/Natanya-DaCosta/api-final-project/get.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+     	"Cost":9.50
+  	}
+   }
+}
+```
+  
+### Properties Table for GET Response Code Snippet  
+The GET response consists of various properties.  These properties are listed below according to their objects. 
+#### JSON Objects 1: **orderNum and timestamp**  
+
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Natanya-DaCosta/api-final-project/get.md:46]: Fot GET responses, you do not need a value column. The value will change depending on what actions are done and what values are in the database. 

Please remove this column

**Review by Laura-Novich-OBW — CHANGES_REQUESTED** [review]
small change to the GET file, keep going you got this!!

**Comment by NatanyaDC** [comment]
Made requested change. Took out the value column in the table of the Get Response.

**Review by lsnovich — APPROVED** [review]

**Comment by lsnovich** [comment]
Natanya, nice job!

**Review by Laura-Novich-OBW — APPROVED** [review]

**Comment by Laura-Novich-OBW** [comment]
Please submit to Google Classroom

**Pull request merged**